index.html

```diff
@@ -274,6 +274,105 @@ <h4 class="stack-title">Skills</h4>
           </div>
         </div>
       </section>
+
+      <!-- contact section -->
+      <div id="contact-section">
+        <h3>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h3>
+        <form
+          id="contact-form"
+          action="https://formspree.io/f/xoqrpjkd"
+          method="post"
+        >
+          <fieldset>
+            <input
+              type="text"
+              id="name"
+              placeholder="Full Name"
+              maxlength="30"
+              required
+            />
+            <input
+              type="email"
+              id="email"
+              placeholder="Email Address"
+              required
+            />
+            <textarea
+              name="message"
+              id="message"
+              cols="10"
+              rows="10"
+              maxlength="500"
+              placeholder="Enter Text Here"
+              required
+            ></textarea>
+          </fieldset>
+          <button type="submit" name="button">Get in touch</button>
+        </form>
+
+        <div id="social-media-links">
+          <ul id="contact-links2">
+            <li class="contact-link">
+              <a
+                href="https://twitter.com/ceemos_dev"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <i class="fab fa-twitter-square"></i
+              ></a>
+            </li>
+
+            <li class="contact-link">
+              <a
+                href="https://www.linkedin.com/in/suleiman-olaide-97689b154/"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <i class="fab fa-linkedin"></i
+              ></a>
+            </li>
+
+            <li class="contact-link">
+              <a
+                href="https://github.com/Ceemos96"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <i class="fab fa-github-square"></i>
+              </a>
+            </li>
+
+            <li class="contact-link">
+              <a
+                href="https://medium.com/@suleimanxxolaide"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <i class="fab fa-medium"></i
+              ></a>
+            </li>
+
+            <li class="contact-link">
+              <a
+                href="https://github.com/Ceemos96"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <i class="fab fa-angellist"></i>
+              </a>
+            </li>
+          </ul>
+        </div>
+
+        <div id="contact-email">
+          <a href="#" target="_blank" rel="noopener noreferrer">
+            Olaide.suleiman96@gmail.com
+          </a>
+        </div>
```

Comment:
- [ ] Kindly follow the font family for the paragraph as per the Figma design.
- [ ] Kindly make sure to add the social media icons below the button as per the Figma design. As you can see currently the social media icons are not displayed.
- [ ] Kindly follow the font size for the email address written below as per the Figma design.
- [ ] Kindly make sure to add the background image at the bottom right side as per the Figma design.

Your design:
<img width="288" alt="Screenshot 2022-04-26 at 12 26 24 PM" src="https://user-images.githubusercontent.com/70269507/165270307-5d9f4050-59f2-4a5e-858a-71bd59d833db.png">
Figma design:
<img width="220" alt="Screenshot 2022-04-26 at 12 26 59 PM" src="https://user-images.githubusercontent.com/70269507/165270318-ded9f2ea-b4e3-4370-bbc2-370c7e30d45c.png">